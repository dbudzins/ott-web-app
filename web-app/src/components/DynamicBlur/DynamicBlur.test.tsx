import React from 'react';
import { render } from '@testing-library/react';

import DynamicBlur from './DynamicBlur';

describe('<DynamicBlur>', () => {
  test('renders and matches snapshot', () => {
    const { container } = render(<DynamicBlur url="/" />);

    expect(container).toMatchSnapshot();
  });
});
