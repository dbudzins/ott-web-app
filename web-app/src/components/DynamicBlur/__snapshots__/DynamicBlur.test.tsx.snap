// Vitest Snapshot v1

exports[`<DynamicBlur> > renders and matches snapshot 1`] = `
<div>
  <div
    class="BlurBackground"
    style="transition: opacity 1s ease-in-out;"
  />
  <div
    class="BlurBackground"
    style="transition: opacity 1s ease-in-out;"
  />
</div>
`;
